src/libraries/System.Threading.Channels/tests/ChannelClosedExceptionTests.netcoreapp.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Threading.Channels.Tests
     public partial class ChannelClosedExceptionTests
     {
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/37648", TestPlatforms.Browser)]
+        [PlatformSpecific(~TestPlatforms.Browser)] // BinaryFormatter not supported in wasm
```

Comment:
I think I'd prefer a `PlatformDetection.IsBinaryFormatterSupported` property for this and using ConditionalFact instead.

Reply:
Hmm ok, makes sense.

Reply:
Done.